client-projects/the-lumery/scss/button.scss

```diff
@@ -0,0 +1,186 @@
+    transform: scale(1.1);
+  }
+}
+
+// in a primary button, a yellow background div is covered with its black ::after
+// pseudoelement on hover.
+.btn__bg--primary {
+  background-color: var(--color-yellow);
+
+  // black bg
+  &::after {
+    content: "";
+    grid-area: button;
+    display: block;
+    border-radius: var(--border-radius);
+    width: 100%;
+    height: 100%;
+    background-color: var(--color-black);
+    transform-origin: center;
+    transform: scaleX(0);
+    transition: transform 300ms var(--easing);
+  }
+
+  // black background expands out horizontally from centre
+  .btn:hover &::after,
+  .card__link:hover &::after,
+  .button--submit:hover + .fake-button-container &::after {
+    transform: scaleX(1);
+  }
+
+  .btn:focus &,
+  .card__link:focus &,
+  .button--submit:focus + .fake-button-container & {
+    outline-color: var(--color-black);
+  }
+
+  // the hovered button would disappear on a black background, so:
+  .background-color-black .btn:hover & {
+    border: 1px solid var(--color-yellow);
+  }
+  .background-color-black .btn:focus & {
+    outline: 2px solid var(--color-white);
+  }
+}
+
+// in a secondary button, there's no background, just a contrasting border
+.btn__bg--secondary {
+  border-width: 1px;
+  border-style: solid;
+  border-color: var(--color-black);
+
+  .btn:focus & {
+    border-color: var(--color-yellow);
+  }
+  input:focus-visible + .fake-button & {
+    border-color: var(--color-white);
+  }
+
+  .background-color-black & {
+    border-color: var(--color-white);
+  }
+
+  .background-color-black .btn:focus & {
+    outline: var(--color-yellow);
+  }
+}
+
+.btn__text {
+  grid-area: button;
+  justify-self: center;
+  z-index: 1;
+  padding-top: 0.625rem;
+  padding-bottom: 0.625rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  text-align: center;
+  color: var(--color-black);
+  font-family: "Metric", sans-serif;
+  font-size: 1.25rem;
+  line-height: 1.25;
+  @include mobile-portrait-down {
+    font-size: 1rem;
+  }
+}
+
+.btn__text--primary {
+  transition: color 300ms linear;
+  .btn:hover &,
+  .card__link:hover &,
+  .button--submit:hover + .fake-button-container & {
+    color: var(--color-white);
+  }
+  .btn:active &,
+  .card__link:active &,
+  .button--submit:active + .fake-button-container & {
+    color: var(--color-yellow);
+  }
+}
+
+// sometimes the 'fake' button components need to be wrapped for layout purposes,
+// we don't pointer events on fake buttons, (they react to interaction on hidden
+// parents or siblings) so they should be removed here too.
+.fake-button-container {
+  pointer-events: none;
+}
+
+// filter buttons on the Projects page
+
+.filter__button-list .button--grouped {
+  flex-grow: 0;
+  flex-shrink: 0;
+  @include mobile-landscape-down {
+    flex-basis: calc(33% - 1rem);
+  }
+  @include mobile-portrait-down {
+    flex-basis: calc(50% - 0.75rem);
+  }
+}
+
+// overwrite secret Webflow 'w-' styles
```

Comment:
🤫

Reply:
bizarre stuff happens on this project all the time, and it's because Webflow is 'helpfully' making a bunch of opinionated changes to the CSS in the background.